src/open_inwoner/utils/tests/test_timeline_logger.py

```diff
@@ -97,40 +97,6 @@ def test_user_does_not_have_add_permission(self):
         response = self.app.get(url, user=self.user, expect_errors=True)
         self.assertEquals(response.status_code, 403)
 
-    def test_user_does_not_have_change_permission(self):
-        add_url = reverse("admin:accounts_contact_add")
-        add_form = self.app.get(add_url, user=self.user).forms.get("contact_form")
-        add_form["first_name"] = self.contact.first_name
-        add_form["last_name"] = self.contact.last_name
-        add_form["email"] = self.contact.email
-        add_form["created_by"] = self.user.id
-        add_form.submit()
-        log_entry = TimelineLog.objects.first()
-        log_url = reverse(
-            "admin:timeline_logger_timelinelog_change",
-            kwargs={"object_id": log_entry.id},
-        )
-        log_form = self.app.get(log_url, user=self.user).forms["timelinelog_form"]
-        log_form["object_id"] = 29
-        response = log_form.submit(expect_errors=True)
-        self.assertEquals(response.status_code, 403)
-
-    def test_user_does_not_have_delete_permission(self):
-        add_url = reverse("admin:accounts_contact_add")
-        add_form = self.app.get(add_url, user=self.user).forms.get("contact_form")
-        add_form["first_name"] = self.contact.first_name
-        add_form["last_name"] = self.contact.last_name
-        add_form["email"] = self.contact.email
-        add_form["created_by"] = self.user.id
-        add_form.submit()
-        log_entry = TimelineLog.objects.first()
-        log_url = reverse(
-            "admin:timeline_logger_timelinelog_delete",
-            kwargs={"object_id": log_entry.id},
-        )
-        response = self.app.post(log_url, user=self.user, expect_errors=True)
-        self.assertEquals(response.status_code, 403)
-
```

Comment:
I would leave these tests in. I would even make the change that it is a superuser and not a normal user that is not allowed to change or delete.